datajoint/connection.py

```diff
@@ -93,7 +94,7 @@ def connect(self):
             warnings.filterwarnings('ignore', '.*deprecated.*')
             self._conn = client.connect(init_command=self.init_fun,
                                         sql_mode="NO_ZERO_DATE,NO_ZERO_IN_DATE,ERROR_FOR_DIVISION_BY_ZERO,"
-                                                 "STRICT_TRANS_TABLES,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION",
+                                                 "STRICT_TRANS_TABLES,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION", charset="utf8",
```

Comment:
Why not let charset be a parameter to be passed into `dj.conn`? I don't like this kind of hard coding. In fact this probably should be part of `conn_info` with default value of `utf8`.

Reply:
making it part of `dj.config` ...

Reply:
If you are going to do this, there has to be associated handling of `dj.config` where if the option is not given then it defaults to `utf8`.

Reply:
We should consider making it possible to set different connection character set for each DataJoint connection. I'll likely follow up with such change, but I'm fine with this current change.